### ISSUE TITLE
Expand RSS Feed to full article

### ISSUE THREAD

**Issue by onlydole** [cncf/cncf.io#645]
The CNCF website RSS feed is only showing the first 400 or so words with a "source" link to read further. Would you be able to expand this to offer the full article for a better RSS viewing experience in Feedly and other RSS readers?

Example: 

<img width="785" alt="Screenshot 2023-01-19 at 2 11 50 PM" src="https://user-images.githubusercontent.com/117500/213573771-e852343f-c74c-4835-acfb-6205409c860c.png">

Proposal: 

<img width="798" alt="Screenshot 2023-01-19 at 2 12 57 PM" src="https://user-images.githubusercontent.com/117500/213573963-243a14c7-29f6-4149-b841-b77b60af9878.png">

**Comment by thetwopct** [comment]
I've looked in to this, and we have WordPress setup already to offer the full post in the RSS feed:

![2023-02-01-001145@2x](https://user-images.githubusercontent.com/10615884/215833538-6254d139-c87e-4fc4-80d2-4af519db2dc4.jpg)

However it looks like our SEO plugin is being "useful" by limiting the posts to 400 characters!

![2023-02-01-001238@2x](https://user-images.githubusercontent.com/10615884/215833782-66cfba15-feab-4200-b622-97c3e6dd5795.jpg)

I have turned off the "Convert feed entries into excerpts?" option, however after clearing cache and refreshing the feed its still showing the excerpt. 

Will have to investigate some more.

**Comment by cjyabraham** [comment]
@thetwopct the plugin author is usually quick to respond if you want to [write him about it](https://wordpress.org/support/plugin/autodescription/).

**Comment by thetwopct** [comment]
I tried this again with both options turned off - "Convert feed entries to excerpts" and "Add link to source below the feed entry content" and now the full article is shown. 

I had to delete the feed from my reader and re subscribe to get previous posts to show in full length, but new posts should work properly for all subscribers.

We should monitor if the feed is abused by other sites as this could impact our own sites SEO score.

We should note in the analytics report for February that we made this change - there is now no need for RSS subscribers to visit the CNCF site to read a post so this will impact traffic. @cjyabraham
